packages/commons/src/utils/constants.ts

```diff
@@ -1,4 +1,8 @@
+import { BigNumber } from "ethers";
```

Comment:
why do we need this import?

Reply:
mistake [e0c8edf](https://github.com/Voltz-Protocol/v2-off-chain-monorepo/pull/34/commits/e0c8edf872ac0156572fc067bdb129d70fc36e1d)